.github/workflows/jgiven_branch_and_pr_build.yml

```diff
@@ -12,6 +12,8 @@ jobs:
         java: [ '11', '13']
         junit: ['4.12','4.13']
     steps:
+      - name: Ensure firefox is installed
+        run: sudo apt-get install firefox
```

Comment:
Will the sudo step work or could it crash due to the lack of credentials/permissions?

Reply:
we generally have root and installation permissions on the ci runners. (We also install jeckyll for releases and sbt stuff for our tests).
In fact, it already worked: https://github.com/TNG/JGiven/actions/runs/4465801439/jobs/7843220117

Reply:
Then everything is fine ;)